### DISCUSSION TITLE
Automatic email verification and Forget password Issue in Wiki 2.2.51

### DISCUSSION THREAD

**Discussion by dinesh121991** [requarks/wiki#3404] [Help / Questions]
**Question**
We followed the wiki.js official documentation to set up the wiki, and it doe not work properly working for following features.

1) We set up self-registration with email verification. Test email is working properly with the SMTP settings in Mail, and an automatic verification email is sent with the proper link. We get the error when the user clicks the verification link. You can find the screenshot and the verification-link format when the user clicks the verification link.
https://DomainName/verify/Xmsuv0a17DgKoeBL95u4M
Configurations followed as below:
https://docs.requarks.io/en/auth/local

![wiki-verification-issue](https://user-images.githubusercontent.com/7029820/87022820-4f40d600-c1d7-11ea-85d0-d0dcafb990d6.png)

2) Forget password is not sending email to the corresponding person email, and it is saying "Coming soon" as shown in the screenshot below. Forgot password is in the link as same as login page - https://DomainName/login.

![wiki-reset-password-issue](https://user-images.githubusercontent.com/7029820/87022816-4e0fa900-c1d7-11ea-86d8-6b098599be62.png)

**Host Info (please complete the following information):**
OS: Ubuntu 18.04
Wiki.js version: 2.2.51
Database engine: postgres 12.2
Nginx is the webserver: nginx version: nginx/1.14.0 (Ubuntu)

**Comment by dinesh121991** [comment]
Hi @Smankusors,
Any update on it. Is it a bug or configuration issues?

**Comment by Smankusors** [comment]
For reset password, it's coming soon. 😉

While you waiting, you can do it directly on the database. Read more about it [here](https://docs.requarks.io/troubleshooting#how-to-manually-reset-the-admin-password)

---

But for the email verification, I have no idea. I'm just volunteering here (not Requarks employee). But unfortunately I have no time to test this out. Sorry. 😥

**Comment by craviee** [comment]
@dinesh121991 I just had the same issue that you had regarding the verification link and I think I can help.
I had this issue and it is because I didn't set a group to a user that just signup.
Every user should have at least one group.
